### ISSUE TITLE
Add logits

### ISSUE THREAD

**Issue by chase-eck** [chase-eck/fsp_purchase#29]
Turns out felm doesn't play nice with logits

probably should use a conditional logit, check out the clogit function from the survival package:
https://stats.stackexchange.com/questions/2134/r-package-for-fixed-effect-logistic-regression
